### DISCUSSION TITLE
v1.0.0

### DISCUSSION THREAD

**Discussion by nekofar** [KartanHQ/intellij-tauri#24] [Announcements]
### Bug Fixes
- Solve some minor issues and update dependencies

### Features
- Implement directory project generator

[1.0.0]: https://github.com/KartanHQ/intellij-tauri/commits/v1.0.0

<hr /><em>This discussion was created from the release <a href='https://github.com/KartanHQ/intellij-tauri/releases/tag/v1.0.0'>v1.0.0</a>.</em>

**Comment by ruffhggjhgjh-hash** [comment]
Hello sir
